gsdk/src/rpc.rs

```diff
@@ -185,4 +188,30 @@ impl Api {
             .await
             .map_err(Into::into)
     }
+
+    /// gear_calculateReplyForHandle
+    pub async fn calculate_reply_for_handle(
```

Comment:
Do we need a param similar to this?
https://github.com/gear-tech/gear/blob/0c429b0a620d1effd9cc0d986748463dd7d00406/gsdk/src/rpc.rs#L86

Reply:
No, there is no such parameter

Reply:
@DennisInSky api already exist on node, so this impl is just access-entry so couldn't impl any custom additional logic. Speaking of your idea it was previously decided to keep api as much similar to real tx sending as possible

Reply:
I guess, this could be resolved